src/clients/eos-sample-pmc

```diff
@@ -0,0 +1,210 @@
+#! /bin/bash
```

Comment:
This file seems to have been added erroneously.

Reply:
Yes my mistake, thanks!